managed/services/grafana/auth_server_test.go

```diff
@@ -292,6 +292,7 @@ func TestServerClientConnection(t *testing.T) {
 		assert.Nil(t, authError)
 	})
 
+	// Beware: Five or more wrong tries will lock user with error message: "Invalid user or password".
```

Comment:
let's change username here to `admin1`, error message should be the same.

Reply:
This is not happening when tests are triggered on github. It can happen when you run them locally multiple times (like -count=5 etc). I added this to not forget this could happen since it is not clear from the error message. Should I keep this comment or remove it?